src/BotiumBotmockExporter.ts

```diff
@@ -0,0 +1,443 @@
+import "dotenv/config";
+import {BaseExporter, Resources, FileWriter, Kind, ProjectReference} from "@botmock/export";
```

Comment:
With the options that I mentioned at tsconfing.json you get in this file some unused import and variable.